dev-docs/development-process.md

```diff
@@ -89,6 +89,14 @@ To run the Taskcluster UI:
 * In a shell window for taskcluster-web-server:
   * Make sure Postgres is set up as described above.
   * Set both `READ_DB_URL` and `WRITE_DB_URL` to the URL for your postgres server, as described above.
+    For example:
+    ```sh
+    export READ_DB_URL=$TEST_DB_URL
+    ```
+    ```sh
+    export WRITE_DB_URL=$TEST_DB_URL
+    ```
```

Comment:
Let's combine the two into a single block:

```suggestion
    ```sh
    export READ_DB_URL=$TEST_DB_URL
    export WRITE_DB_URL=$TEST_DB_URL
    ```
```